ui/admin/app/components/form/host-catalog/azure/index.hbs

```diff
@@ -145,19 +142,26 @@
     @value={{@model.subscription_id}}
     @label={{t 'form.subscription_id.label'}}
     @helperText={{t 'form.subscription_id.help'}}
-    @linkText={{t 'form.subscription_id.link-text'}}
-    @link='https://learn.hashicorp.com/tutorials/boundary/cloud-host-catalogs#set-up-cloud-hosts'
+    @linkText={{t 'titles.learn-more'}}
     readonly={{false}}
   />
 
   <form.input
     @name='client_id'
     @type='client_id'
     @value={{@model.client_id}}
-    @label={{t 'form.provider.client.label'}}
-    @helperText={{t 'form.provider.client.help'}}
-    @linkText={{t 'form.provider.client.link-text'}}
-    @link='https://learn.hashicorp.com/tutorials/boundary/cloud-host-catalogs#set-up-cloud-hosts'
+    @label={{t 'form.azure_client_id.label'}}
+    @helperText={{t 'form.azure_client_id.help'}}
+    @linkText={{t 'titles.learn-more'}}
+    readonly={{false}}
+  />
+
+  <form.input
+    @name='secret_value'
+    @type='secret_value'
+    @label={{t 'titles.secret'}}
```

Comment:
Should this be from a form label translation?

Reply:
yeah it makes sense as a form:label and noticed we have a filter present in the form. 
thoughts on making this as the below?
```
host_set_filter: 
  label: Filter
  help:

Reply:
Ah put it under the resource then.

Reply:
updating!